src/main/java/io/antmedia/filter/JWTFilter.java

```diff
@@ -45,7 +47,15 @@ public void doFilter(ServletRequest request, ServletResponse response, FilterCha
 			((HttpServletResponse) response).sendError(HttpServletResponse.SC_FORBIDDEN, "Application is getting initialized");
 			return;
 		}
-		if(!appSettings.isJwtControlEnabled() || (httpRequest.getHeader(JWT_TOKEN_HEADER) != null && checkJWT(httpRequest.getHeader(JWT_TOKEN_HEADER)))) {
+
+		String jwtToken = httpRequest.getHeader(JWT_TOKEN_AUTHORIZATION_HEADER);
```

Comment:
It checks against "Authorization"  (`JWT_TOKEN_AUTHORIZATION_HEADER`) and in the second line it makes the things for "Bearer"  (`JWT_TOKEN_AUTHORIZATION_HEADER_BEARER_PREFIX`) header.

Reply:
Just ignore this message. It's my mistake I remember something wrong